### DISCUSSION TITLE
Is the documentation page for Neural Prophet down?

### DISCUSSION THREAD

**Discussion by Msaleh87** [ourownstory/neural_prophet#876] [Q&A - get help using NeuralProphet] [answered]
I tried to access the documentation page multiple times, is it being updated or there is a new link to access it?
Thanks

**Comment by Msaleh87** [comment, marked as answer]
I think the links has changed. I googled "Neural Prophet web documentation" instead of "Neural Prophet documentation" and now it started to give me working links: 
https://neuralprophet.com/
https://neuralprophet.com/contents.html

**Comment by ourownstory** [comment]
Hi @Msaleh87 It was indeed down for a few hours due to failed updates. Thank you for letting us know.
Sorry for causing any issues!
